packages/flutter_tools/gradle/flutter.gradle

```diff
@@ -168,6 +167,42 @@ class FlutterPlugin implements Plugin<Project> {
         }
     }
 
+    /**
+     * Adds suitable flutter.jar compile dependencies to the specified buildType.
+     *
+     * Note: The BuildType DSL type is not public, and is therefore just Object here.
+     */
+    private void addFlutterJarCompileDependency(Project project, Object buildType) {
```

Comment:
I believe Groovy-style is to leave the type out when you can't/don't want to specify it for arguments (and use `def` when declaring untyped variables). Section 3 of http://groovy-lang.org/style-guide.html. :)